src/bindings/c/include/openvino/c/ov_model.h

```diff
@@ -32,44 +32,128 @@ typedef struct ov_model ov_model_t;
 OPENVINO_C_API(void) ov_model_free(ov_model_t* model);
 
 /**
- * @brief Get the outputs of ov_model_t.
+ * @brief Get a const input port of ov_model_t.
```

Comment:
It works only for model with single input, I think we need to add information about it to the documentation.

Reply:
@RICKIE777 please add additional information on this API, which only support single input model.

Reply:
Update it in new commit. Please review.